.github/workflows/delete-env.yaml

```diff
@@ -0,0 +1,19 @@
+name: Delete deployed environment
+
+on:
+  pull_request:
+    types: [closed]
+
+# Cancel previous runs of the same workflow and PR number or branch/tag
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  call-workflow:
+    # TODO update the branch reference before merging the PR
+    uses: dhis2-sre/gha-workflows/.github/workflows/im-delete-env.yaml@DEVOPS-262
```

Comment:
Is it true that right now we always run `skaffold delete` on closing a PR other than a dependabot PR? Are you doing this so that when someone adds the deploy label and removes it again we still clean up when the PR is closed?

Reply:
Yep, exactly right. That seemed like the easiest option for making sure to clean up after a PR regardless of whether it was just closed, merged and whether something actually got deployed, etc.